[2023-07-22T05:54:12.014+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-18T00:00:00+00:00 [queued]>
[2023-07-22T05:54:12.051+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-18T00:00:00+00:00 [queued]>
[2023-07-22T05:54:12.054+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:54:12.057+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-07-22T05:54:12.066+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:54:12.172+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-07-18 00:00:00+00:00
[2023-07-22T05:54:12.211+0000] {standard_task_runner.py:55} INFO - Started process 5155 to run task
[2023-07-22T05:54:12.238+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2023-07-18T00:00:00+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpp6wb5thg']
[2023-07-22T05:54:12.247+0000] {standard_task_runner.py:83} INFO - Job 39: Subtask create_table
[2023-07-22T05:54:12.536+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2023-07-18T00:00:00+00:00 [running]> on host ef71d3f52879
[2023-07-22T05:54:13.039+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-07-18T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-18T00:00:00+00:00
[2023-07-22T05:54:13.086+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-07-22T05:54:13.128+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2023-07-22T05:54:13.142+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-07-22T05:54:13.182+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230718T000000, start_date=20230722T055412, end_date=20230722T055413
[2023-07-22T05:54:13.386+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-07-22T05:54:13.576+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-07-22T05:56:55.167+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-18T00:00:00+00:00 [queued]>
[2023-07-22T05:56:55.187+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-18T00:00:00+00:00 [queued]>
[2023-07-22T05:56:55.187+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:56:55.187+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-07-22T05:56:55.187+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:56:55.208+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-07-18 00:00:00+00:00
[2023-07-22T05:56:55.225+0000] {standard_task_runner.py:55} INFO - Started process 5453 to run task
[2023-07-22T05:56:55.258+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2023-07-18T00:00:00+00:00', '--job-id', '102', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpln3mt2ss']
[2023-07-22T05:56:55.274+0000] {standard_task_runner.py:83} INFO - Job 102: Subtask create_table
[2023-07-22T05:56:55.416+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2023-07-18T00:00:00+00:00 [running]> on host ef71d3f52879
[2023-07-22T05:56:55.599+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-07-18T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-18T00:00:00+00:00
[2023-07-22T05:56:55.622+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-07-22T05:56:55.634+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2023-07-22T05:56:55.638+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-07-22T05:56:55.667+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230718T000000, start_date=20230722T055655, end_date=20230722T055655
[2023-07-22T05:56:55.779+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-07-22T05:56:55.931+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
